op-chain-ops/cmd/celo-migrate/main.go

```diff
@@ -0,0 +1,462 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"math/big"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/ethereum-optimism/optimism/op-chain-ops/genesis"
+	"github.com/ethereum-optimism/optimism/op-service/jsonutil"
+	oplog "github.com/ethereum-optimism/optimism/op-service/log"
+	"github.com/ethereum-optimism/optimism/op-service/predeploys"
+	"github.com/holiman/uint256"
+	"github.com/mattn/go-isatty"
+
+	"github.com/urfave/cli/v2"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core"
+	"github.com/ethereum/go-ethereum/core/rawdb"
+	"github.com/ethereum/go-ethereum/core/state"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/ethereum/go-ethereum/ethdb"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/params"
+	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/ethereum/go-ethereum/trie"
+	"github.com/ethereum/go-ethereum/triedb"
+)
+
+var (
+	deployConfigFlag = &cli.PathFlag{
+		Name:     "deploy-config",
+		Usage:    "Path to deploy config file",
+		Required: true,
+	}
+	l1DeploymentsFlag = &cli.PathFlag{
+		Name:  "l1-deployments",
+		Usage: "Path to L1 deployments JSON file",
+	}
+	l1RPCFlag = &cli.StringFlag{
+		Name:  "l1-rpc",
+		Usage: "RPC URL for an Ethereum L1 node",
+	}
+	l2AllocsFlag = &cli.PathFlag{
+		Name:  "l2-allocs",
+		Usage: "Path to L2 genesis allocs file",
+	}
+	outfileL2Flag = &cli.PathFlag{
+		Name:  "outfile.l2",
+		Usage: "Path to L2 genesis output file",
+	}
+	outfileRollupFlag = &cli.PathFlag{
+		Name:  "outfile.rollup",
+		Usage: "Path to rollup output file",
+	}
+	dbPathFlag = &cli.StringFlag{
+		Name:     "db-path",
+		Usage:    "Path to database",
+		Required: true,
+	}
+	dbCacheFlag = &cli.IntFlag{
+		Name:  "db-cache",
+		Usage: "LevelDB cache size in mb",
+		Value: 1024,
+	}
```

Comment:
Do we really need this parameter? This script runs in a few seconds, right? So it seems there's not really a need to let users tweak performance variables.